include/crm/pengine/pe_types.h

```diff
@@ -116,6 +116,9 @@ enum pe_find {
 
 #  define pe_flag_quick_location        0x00100000ULL
 #  define pe_flag_sanitized             0x00200000ULL
+
+#  define pe_flag_print                 0x00400000ULL
+//! \deprecated
```

Comment:
pcmk__is_daemon might be suitable for this, but all the other tools will hit !pcmk__is_daemon as well so make sure they don't suddenly become noisy :)

Reply:
I haven't seen any extra noisiness yet, but I'll stay on the lookout.

Reply:
I haven't seen any extra noisiness yet, but I'll stay on the lookout.